### DISCUSSION TITLE
Improve cursor highlight

### DISCUSSION THREAD

**Discussion by rampatra** [Softal-io/presentify-community#6] [Ideas]
- Slider to change cursor highlight size
- More styles
- Minor bug fixes where the highlight appears at the bottom of the screen
- Remember whether cursor highlight was on before annotation started and start it automatically after annotation is done

**Comment by withtlc** [comment]
The "Click" with the cursor shrinks the ring/circle...  which is great for when one is actively showing a presentation... but if one does a screen grab - one can't tell one is "clicking"...     A different Feedback graphic should be used when a click is made (sort of a splat image.. or multiple concentric circles whose center is the cursor tip..

**Reply by rampatra** [reply]
What do you mean by screen grab? Is it a screenshot? If yes, how is it possible to show animation in an image? If you mean screen recorded as a video then yes, the animation (you recommended above) would be better.

**Comment by rampatra** [comment]
All the points (mentioned in this ticket) are now addressed in the latest version of Presentify v6.0.0

![app-store-cursor-highlight](https://github.com/user-attachments/assets/44ab34ab-6f0b-4b2e-916f-4ef41d09a92b)
